app/locales/sv.json

```diff
@@ -44,16 +51,33 @@
   "file": "Fil",
   "dismiss": "AVFÄRDA",
```

Comment:
Is this used anywhere? I couldn't find it. I suspect a less 'formal' word could be used but I'd have to see the context...

Reply:
You mean "avfärda". I think it's used when you add annotations to the graph and then want to discard them

Reply:
yeah, maybe 'ta bort' would work in this case. I can't get annotations to work though to see what it looks like

Reply:
Lets leave it until we encounter it